portfolio/src/main/java/com/google/sps/servlets/BlogCommentsServlet.java

```diff
@@ -44,6 +41,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the input from the form.
+    Date date = new Date();
+    String text = request.getParameter("comment-text");
+
+    // Upload comment 
+    uploadComment(date, text);
+
+    // Redirect back to the HTML page.
+    response.sendRedirect("/blog.html");
+  }
+
+  private void uploadComment(Date date, String text) {
+
+      BlogComment c = new BlogComment(date, text);
```

Comment:
We need better variable names :D

Reply:
I hear you! I'll switch to a more verbose variable name.